index.html

```diff
@@ -2224,8 +2224,10 @@ <h2 id="utility">Utility Functions</h2>
         interpolated verbatim, an <b>escape</b> regex to match expressions that should
         be inserted after being HTML-escaped, and an <b>evaluate</b> regex to match
         expressions that should be evaluated without insertion into the resulting
-        string. You may define or omit any combination of the three.
-        For example, to perform
+        string. Note that if part of your template matches more than one of these
+        regexes, the first of the following priority order will be applied:
+        <b>escape</b>, <b>interpolate</b>, or <b>evaluate</b>. You may also define or 
+        omit any combination of the three. For example, to perform
```

Comment:
Would you be OK with the following slight changes of language?

```suggestion
        string. Note that if part of your template matches more than one of these
        regexes, the first will be applied by the following order of priority:
        (1) <b>escape</b>, (2) <b>interpolate</b>, (3) <b>evaluate</b>. You may
        define or omit any combination of the three. For example, to perform
```

Reply:
Sure, fine with me. Thanks for taking the time to look at it.